numpy/core/fromnumeric.py

```diff
@@ -816,6 +817,11 @@ def argpartition(a, kth, axis=-1, kind='introselect', order=None):
     >>> np.array(x)[np.argpartition(x, 3)]
     array([2, 1, 3, 4])
 
+    >>> x = np.array([[3, 4, 2], [1, 3, 1]])
+    >>> np.take_along_axis(x, np.argpartition(x, kth=1, axis=-1), axis=-1) # same as np.partition(x, kth=1)
```

Comment:
Let's split this too:
```suggestion
    Multi-dimensional array:

    >>> index_array = np.argpartition(x, kth=1, axis=-1)
    >>> np.take_along_axis(x, index_array , axis=-1)  # same as np.partition(x, kth=1)
```